examples/kafka/kafka-ephemeral.yaml

```diff
@@ -5,13 +5,13 @@ metadata:
 spec:
   kafka:
     version: 2.4.0
-    replicas: 3
+    replicas: 1
```

Comment:
This doesn't look right. I guess you maybe didn't wanted to commit this?

Reply:
Damn, I was sure I reverted changes in this file. Thanks